### PR TITLE
Remove status-utf8 option from solarized themes

### PR DESCRIPTION
**Pull request by mtrolley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Here I assume that you want to install themes into directory
 
 2.  Clone this repository:
     
-        git clone git@github.com:dmitry-kabanov/tmux-statusline-themes.git
+        git clone git@github.com:trolley/tmux-statusline-themes.git
```
Comment (dmitry-kabanov) [inline README.md]: Nice catch of outdated repo name (`kabanovdmitry`), thanks.
But I would prefer to keep `dmitry-kabanov` instead of `trolley`.

Comment (mtrolley) [reply README.md]: Whoops, sorry! That was only meant for my fork until you had my PR merged. I'll fix this.

**Comment by trolley** [comment]
Fixed, thank you :)

File: README.md (hunk 1 above)
Comment (mtrolley) [reply README.md]: Fixed, thank you :)

**Review by dmitry-kabanov — CHANGES_REQUESTED** [review]
Could you change `trolley/tmux-statusline-themes.git` to `dmitry-kabanov/tmux-statusline-themes.git`?

**Comment by mtrolley** [comment]
> Could you change `trolley/tmux-statusline-themes.git` to `dmitry-kabanov/tmux-statusline-themes.git`?

I removed that commit. Check the Files Changed and you'll see it's only the removal of the options now.

**Comment by dmitry-kabanov** [comment]
@trolley thank you very much for the pull request!

**Pull request merged**